portfolio/src/main/java/com/google/sps/servlets/DeleteSingleComment.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/delete-single-comment")
+public class DeleteSingleComment extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    long id = Long.parseLong(request.getParameter("id"));
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Key deleteEntry = KeyFactory.createKey("Comment", id);
+    datastore.delete(deleteEntry);
+
```

Comment:
I would remove the two empty lines in this method:
https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style
https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace